modules/apps/content-dashboard/content-dashboard-document-library-impl/src/main/java/com/liferay/content/dashboard/document/library/internal/item/FileEntryContentDashboardItem.java

```diff
@@ -481,6 +494,66 @@ public boolean isViewable(HttpServletRequest httpServletRequest) {
 			_fileEntry, httpServletRequest);
 	}
 
+	private String _getAspectRatio(Locale locale) {
+		if (!FeatureFlagManagerUtil.isEnabled(
+				_fileEntry.getCompanyId(), "LPD-30087")) {
+
+			return null;
+		}
+
+		FileVersion fileVersion;
+
+		try {
+			fileVersion = _fileEntry.getFileVersion();
+		}
+		catch (PortalException portalException) {
+			if (_log.isDebugEnabled()) {
+				_log.debug(portalException);
+			}
+
+			return null;
+		}
```

Comment:
Aren't we hiding possible errors with this? I'd either let the error bubble upwards or log it as an error. This shouldn't be a debug trace.

Reply:
I put debug here because we normally set log traces as debug but I can change it, there is not problem for me to change it.